### ISSUE TITLE
Cannot set a default for enumeration

### ISSUE THREAD

**Issue by martasd** [tOgg1/graphene-django-cud#4]
I have the following field in my Beo model:
```
class Beo(BaseModel):
    ...
    STATUSES = (("UNLOCKED", "Unlocked"), ("LOCKED", "Locked"), ("SIGNED", "Signed"))
    status = CharField(max_length=16, choices=STATUSES, default="UNLOCKED")

```

When using **DjangoCreateMutation**, I get an enumeration for this field as expected:

![image](https://user-images.githubusercontent.com/591911/68109774-b6411900-feeb-11e9-99e7-e555bc5c5731.png)

When I call the mutation without including the **status** field, I get the following error message even though I specify a default in the Beo model:

`In field \"status\": Expected \"BeoStatus!\", found null.",`

**Comment by tOgg1** [comment]
Should be fixed in #4.

**Comment by martasd** [comment]
Great. Thanks!
